### ISSUE TITLE
The link for fashiongen does not work

### ISSUE THREAD

**Issue by Arsmart123** [menardai/FashionGenAttnGAN#8]
As shown in the file: 
Download
You can download the training set (download/train) and the validation set
(download/validation).
But the link does not work....

**Comment by freshfish15** [comment]
Having the same issue. Will there be an update?
